docs/docs/20-usage/35-advanced-yaml-tricks.md

```diff
@@ -0,0 +1,61 @@
+# Advanced YAML tricks
```

Comment:
Could you link this page from the pipeline syntax list somehow like we did it for `secrets`.

Reply:
I dont understand

Reply:
Adding a new paragraph to https://woodpecker-ci.org/docs/usage/pipeline-syntax on the top-level like `services` with a content similar to this: 

> Woodpecker provides to use [YAML anchors & aliases}(https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases) in the pipeline configuration. These can be used to write DRY configs by using variables.
> 
> For more details check the [Advanced YAML syntax docs](https://woodpecker-ci.org/docs/usage/advanced-yaml-syntax).

Reply:
I dont think we should link it there ...

Reply:
Why not wouldn't it be incomplete somehow if we don't link it? 🤔

Reply:
well it's no specific pipeline config thing ... this document yaml spec behaviour

Reply:
I get your point, but it now somehow has a specific position in the config with `variables` now. In addition I think users would search how to use variables quite often as seen in the issues, so linking it here as well could be helpful.

Reply:
@anbraten can you push to this branch to make the specific docs changes?

Reply:
I think the branch is protected, but I can create a PR against it.

Reply:
@anbraten **https://github.com/6543-forks/woodpecker/invitations**